### ISSUE TITLE
AS 3.7.1: "autosubmit create expid" is not progressing

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#180]
In GitLab by @Eleftheriaexarchou on Aug 24, 2016, 15:59

Hi, 

I test AS 3.7.1 but : when I create an experiment, it is stuck in the stage below for several minutes. If I revert back to the older version 3.6.0, the *autosubmit create* works just fine in few seconds  

```      
eexarcho@ bscearth304:/esnas/autosubmit/a09w/conf$ autosubmit create a09w 
Preparing .lock file to avoid multiple instances with same expid.

Checking configuration files...
autosubmit_a09w.conf OK
platforms_a09w.conf OK
jobs_a09w.conf OK
expdef_a09w.conf  OK
Configuration files OK

Using project folder: /esnas/autosubmit/a09w/proj
Loading parameters...

Creating joblist...
Creating jobs...
Adding dependencies...
Removing redundant dependencies...
```       

@ jlope2

**Comment by kinow** [comment]
In GitLab by @ editomas on Sep 13, 2016, 11:01

Good to know that, thanks. In the meantime, which AS version would be safe to switch to? e.g., to use auto-nmmb/develop-da branch for longer than two days?
@jordicuadrado

**Comment by kinow** [comment]
In GitLab by @ editomas on Sep 15, 2016, 19:00

@jordicuadrado your a51b887a works with AS 3.6.0. So, this issue is no longer a problem for auto-nmmb/develop-da: we can continue with the default version. thanks!

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Oct 13, 2016, 23:35

Changed title: **AS 3.7.1: &quot;autosubmit create expid&quot; is not progressing{-  -}** → **AS 3.7.1: &quot;autosubmit create expid&quot; is not progressing**

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Oct 13, 2016, 23:35

Milestone removed

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Oct 28, 2016, 09:54

Mentioned in commit 83f8c829395b5395e330721e8c936725a5ba97dd

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Dec 23, 2016, 16:35

Hi,

This problem should already be solved/improved in the most recent versions (3.7.7 & 3.7.8) of Autosubmit.

I proceed to close this issue. Open an other one if needed.

Thanks!

**Comment by kinow** [comment]
In GitLab by @jordicuadrado on Oct 17, 2017, 16:21

mentioned in commit auto-monarch@a51b887ada1fcfcfb23f5ec0788ece30b6fbf36d
